pkg/apis/cassandraoperator/v1alpha1/cassandradatacenter.go

```diff
@@ -5,16 +5,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-
 // CassandraDataCenterSpec defines the desired state of CassandraDataCenter
 // +k8s:openapi-gen=true
 type CassandraDataCenterSpec struct {
-	Cluster v1.LocalObjectReference `json:"cluster"`
-	Replicas int32 `json:"replicas"`
-	CassandraImage string `json:"cassandraImage"`
-	SidecarImage string `json:"sidecarImage"`
-	ImagePullPolicy v1.PullPolicy `json:"imagePullPolicy"`
-	ImagePullSecrets []v1.LocalObjectReference `json:"imagePullSecrets"`
+	// Cluster is either a string or v1.LocalObjectReference
+	//Cluster interface{} `json:"cluster,omitempty"`
+	Cluster          string                    `json:"cluster,omitempty"`
+	Nodes            int32                     `json:"nodes"`
```

Comment:
Changed `replicas` -> `nodes`, to fit within Cassandra parlance.

Reply:
Does replicas have a special meaning in K8s tho? E.g. see https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#scale-subresource

Or is this just an internal struct where the name doesn't matter a ton

Reply:
Yet to play around with the `/scale` endpoint stuff, but my general understanding is that `SpecReplicasPath` and `StatusReplicasPath` can point to any field -- it doesn't need to be called `Replicas`.

i.e, instead of `specReplicasPath: .spec.replicas` we could have `specReplicasPath: .spec.nodes`.